packages/expo/package.json

```diff
@@ -71,7 +71,7 @@
     "@types/react-dom": "*",
     "expo-auth-session": "^5.4.0",
     "expo-local-authentication": "^13.5.0",
-    "expo-secure-store": "^12.4.0",
+    "expo-secure-store": "^12.8.1",
```

Comment:
this includes [breaking changes](https://github.com/expo/expo/blob/sdk-51/packages/expo-secure-store/CHANGELOG.md#-breaking-changes-2), but since it's a devDep, we can allow it